modules/installation-uninstall-aws.adoc

```diff
@@ -7,27 +7,87 @@
 
 You can remove a cluster that you installed on Amazon Web Services (AWS).
 
+.Prerequisites
+
+* Deploy an {product-title} cluster.
```

Comment:
Seems a bit obvious to me that you need a cluster in order to delete one!

Reply:
@jianlinliu, do you know if this is the right process to remove a partially-deployed cluster? If this works if the installation process did not complete, I'll remove the prereq. (I haven't had a cluster deployment fail since I got this bug, so I am not sure.)

Reply:
Pls remove the prereq